content/v2.0/backup-restore/backup.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Back up data
+seotitle: Back up data in InfluxDB
+description: >
+  Use the `influx backup` command to back up all data and metadata stored in InfluxDB.
+menu:
+  v2_0:
+    parent: Back up & restore data
```

Comment:
and

Reply:
I'm leaving the ampersand in the menu title (changed it in the page title) to save on line length in the nav.  To tie this to the correct parent, this needs to stay the same.